packages/server/postgres/migrations/1658834062488_createMeetingSeries.ts

```diff
@@ -0,0 +1,33 @@
+import {Client} from 'pg'
+import getPgConfig from '../getPgConfig'
+
+export async function up() {
+  const client = new Client(getPgConfig())
+  await client.connect()
+  await client.query(`
+    CREATE TABLE IF NOT EXISTS "MeetingSeries" (
+      "id" INT GENERATED BY DEFAULT AS IDENTITY PRIMARY KEY,
+      "meetingType" "MeetingTypeEnum" NOT NULL,
+      "title" TEXT NOT NULL,
```

Comment:
-1 could we make this something reasonable like `VARCHAR(255)`? (same for recurrenceRule)
2 reasons:
- prevent a DOS attack like someone adding a 5MB title that causes our DB to slow down. (I think we do this in the application layer, but it's always nice to prevent it at the DB layer, too!)
- give other devs an estimate about how large this field is

Reply:
TIL!  Fixed